README.md

```diff
@@ -0,0 +1,207 @@
+<a name="readme-top"></a>
```

Comment:
Good job adding the readme.  Now, kindly:

- [ ] This command won't work.  Kindly remove it from your instructions:

![image](https://github.com/RitaDee/event_connect_client/assets/57421823/c464f9dd-ca4a-43d1-b4a0-8b55e5a610a1)

![image](https://github.com/RitaDee/event_connect_client/assets/57421823/67b4910c-ef4f-4764-966a-78b70f8272bb)

The correct command is `npm start` which you already have later in your instructions.

- [ ] You state in your instructions how to run frontend tests, but your project does not include any of them:

![image](https://github.com/RitaDee/event_connect_client/assets/57421823/fee6ec12-0fce-490c-a9cb-ae835c4d91a7)

![image](https://github.com/RitaDee/event_connect_client/assets/57421823/9eed2e69-4dc0-4bfe-8e78-d9c3fff463bf)

You have 2 options with this one:  Either write some frontend tests so that the `npm run test` command works, or kindly remove this command/section from your instructions in order to prevent confusion.

That way, your project will be well documented, and it will be ready for prime time. 💪